x/msgfees/types/msgs.go

```diff
@@ -50,9 +55,29 @@ func (msg MsgAssessCustomMsgFeeRequest) ValidateBasic() error {
 	if !msg.Amount.IsPositive() {
 		return errors.New("amount must be greater than zero")
 	}
+	_, err = msg.GetBips()
+	if err != nil {
+		return err
+	}
 	return nil
 }
 
+// GetBips converts the msg RecipientBasisPoints to a uint32 basis point value 0 - 10,000
+func (msg MsgAssessCustomMsgFeeRequest) GetBips() (uint32, error) {
+	if msg.RecipientBasisPoints == "" {
+		return AssessCustomMsgFeeBips, nil
+	}
+	bips, err := strconv.ParseUint(msg.RecipientBasisPoints, 10, 32)
+	if err == nil {
+		if bips > 10_000 {
```

Comment:
`10_000` should probably be a constant `RECIPIENT_MAX_BIPS` or something like that.

Reply:
Come to think of it. Could this be better improved as a default parameter and then we can set min max through params?

Reply:
I'm not sure if we need to have a variable for this.  `RECIPIENT_MAX_BIPS` would imply that we are capping the amount of bips a recipient can have.  Here we are allowing the user of the assess msg to freely set how much they want.  The only possible range of basis points is 0 - 10,000 (0% - 100%).

Reply:
👍